pkg/console/operator/operator.go

```diff
@@ -109,7 +112,8 @@ func NewConsoleOperator(
 		oauthClient:   oauthv1Client,
 		versionGetter: versionGetter,
 		// recorder
-		recorder: recorder,
+		recorder:       recorder,
+		resourceSyncer: resourceSyncer,
```

Comment:
I would expect a config informer name addition below.

Reply:
I do not have another config informer.  The informer is part of the resourceSyncer.

Reply:
@enj - Can you clarify this further? We do have the config informer set just as it is in the cluster auth op.

Reply:
This will probably clarify:
https://github.com/openshift/cluster-authentication-operator/blob/master/pkg/operator2/operator.go#L230

Reply:
Further this line:
```
operator.WithInformer(configMapInformer, operator.FilterByNames(api.OpenShiftConsoleConfigMapName, api.ServiceCAConfigMapName)),

```
Shows that right now in the`openshift-console` namespace, the `informers` only fire on changes to `console-config` & `service-ca`.  If you sync in `my-logo`, `my-logo2`, etc, these will not trigger the informer.  In addition, if `my-logo` is already sync'd into the `openshift-console` namespace, but the user changes the value of the logo within the configmap, this will also NOT be noticed.

It's possible you are seeing sync because you are triggering the loop in other ways.  By the time a typical user is getting to the point of configuring a custom logo, the system will be very idle.  There won't be other events to piggy-back on.  If you happen to be testing with some bash scripts, I'd be cautious about what they are doing that might be triggering the loop in unexpected ways.

Reply:
So, we are assuming a "non-happy" path where the user changes the configmap directly in the `openshift-console` namespace? We should only expect a change when the user changes the configmap in `openshift-config` but I agree we should cover this just in case.  
As a side question, if the user can manipulate the `openshift-console` namespace, why did we bother putting the custom logo config map in the `openshift-config` in the first place?

Reply:
done

Reply:
No, we are not saying the user changes a configmap directly in `openshift-console`.  I think you are misunderstanding the design here.  You have 2 sync loops running with the `syncer`.  

When the `resource syncer` finally decides "yes, this is a logo, copy it over"... our main sync needs to know that the new `custom-logo` configmap even exists & watches is so that it updates (and you cannot trust an accidental update because the sync loop ran for another reason).

Reply:
Make sure you understand how the sync loops communicate.  It is a very small overlap (precisely to ensure our main loop is not triggered too much).